src/test/java/edu/ucsb/cs156/organic/services/SystemInfoServiceImplTests.java

```diff
@@ -29,7 +30,20 @@ void test_getSystemInfo() {
     SystemInfo si = systemInfoService.getSystemInfo();
     assertTrue(si.getSpringH2ConsoleEnabled());
     assertTrue(si.getShowSwaggerUILink());
-    assertEquals("https://github.com/ucsb-cs156/proj-organic", si.getSourceRepo());
+    assertEquals("https://github.com/ucsb-cs156-s24/proj-organic-s24-5pm-4", si.getSourceRepo());
```

Comment:
```suggestion
       assertEquals("https://github.com/ucsb-cs156/proj-organic", si.getSourceRepo());
```

The default value should be `https://github.com/ucsb-cs156/proj-organic"` in the properties files, and you should be able to override it by setting SOURCE_REPO.